in/in_command_test.go

```diff
@@ -1,8 +1,13 @@
 package in_test
 
 import (
+	"archive/tar"
+	"compress/gzip"
 	"fmt"
+	"io"
```

Comment:
This test is at a different level than the rest of this suite/file. Can we mock these things out, or move this to an integration test. Or, preferably, both.

Reply:
I can look at how to refactor the test